homeassistant/components/group/strings.json

```diff
@@ -190,5 +190,55 @@
         "product": "Product"
       }
     }
+  },
+  "services": {
+    "reload": {
+      "name": "Reload",
+      "description": "Reloads group configuration, entities, and notify services from YAML-configuration."
+    },
+    "set": {
+      "name": "Set",
+      "description": "Creates/Updates a user group.",
+      "fields": {
+        "object_id": {
+          "name": "Object ID",
+          "description": "Group ID and part of entity ID."
+        },
+        "name": {
+          "name": "Name",
+          "description": "Name of group."
+        },
+        "icon": {
+          "name": "Icon",
+          "description": "Name of icon for the group."
+        },
+        "entities": {
+          "name": "Entities",
+          "description": "List of all members in the group. Not compatible with 'delta'."
```

Comment:
```suggestion
          "description": "List of all members in the group. Cannot be run in combination with `Add Entities` or `Remove Entities`."
```

Reply:
Should it be `Cannot be used in combination`?